rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/ast/statements.kt

```diff
@@ -84,11 +85,12 @@ interface CompositeStatement {
     val body: List<Statement>
 }
 
-fun List<Statement>.explode(): List<Statement> {
+fun List<Statement>.explode(preserveCompositeStatement: Boolean = false): List<Statement> {
```

Comment:
what is the scope of `explode`?
Which type of statement is `preserveCompositeStatement`?

Reply:
There are two types of statements: Statement and CompositeStatement. 
CompositeStatement is a statement (for example IF) that contains other statements  (children statements contained in the  ELSE, ELSEIF, etc).
The function explode is used to extract the children statements but does not preserve CompositeStatement, for this reason I had to add param preserveCompositeStatement in order to preserve it. This thing has been helpful in the context of dspec inline definition for DO codop.